src/Microsoft.ML.Core/Data/MetadataUtils.cs

```diff
@@ -367,7 +367,7 @@ public static bool IsNormalized(this Schema schema, int col)
         /// of a scalar <see cref="BoolType"/> type, which we assume, if set, should be <c>true</c>.</returns>
         public static bool IsNormalized(this SchemaShape.Column col)
         {
-            Contracts.CheckValue(col, nameof(col));
+            Contracts.Check(col.IsValid, nameof(col));
```

Comment:
>Check [](start = 22, length = 5)

This would be `CheckParam`. Also you will want to provide some sort of message.

Reply:
Sure. We will use `Contracts.CheckParam(col.IsValid, nameof(col), "struct not initialized");`

---
In reply to: [238870227](https://github.com/dotnet/machinelearning/pull/1822#discussion_r238870227) [](ancestors = 238870227)